doc/how-to/initialise.rst

```diff
@@ -90,6 +90,35 @@ Complete the following steps to initialise MicroCloud:
 
 See an example of the full initialisation process in the :ref:`Get started with MicroCloud <initialisation-process>` tutorial.
 
+Excluding MicroCeph or MicroOVN from MicroCloud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question:
```

Comment:
```suggestion
If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question::
```

Sorry, missed that in my previous review. I had it in my first one though. :stuck_out_tongue_closed_eyes: 
You need two colons to introduce a code block.

Same change below as well.